onboarding.md

```diff
@@ -113,49 +180,91 @@ If you will be out of town for work or vacation, note this in the calendar.
 
 ## Meetings
 
-**Scrum:** The scrum is a 10 minute or less meeting that is held on Monday and Friday.
-It is currently scheduled for 10:00 AM on Monday and 1:40 PM on Friday.
-The goal of the scrum is to provide a check in on projects and to keep lab members up to date with their peers.
-In addition to this, we hold a "virtual" scrum/goals list each day.
-Every day, a new issue is posted in the [scrum repository](https://github.com/greenelab/scrum/issues/).
-Lab members are expected to note their daily goals and who, if anyone, is blocking the lab member and who, if anyone, the lab member is blocking.
+**Scrum:** Our team's scrum process involves three components:
+
+    1. An opening scrum meeting at 10:00 AM Monday morning where goals
+       for the week will be laid out, in the form of post-it notes, on
+       the whiteboard in the Greene Lab.
+    2. A closing scrum meeting at 1:40 PM Friday afternoon where team
+       members will reflect on the progress they made on the goals
+       they stated the previous Monday.
+    3. A daily virtual scrum update.
+
+The daily virtual scrum update should include an update to [scrum
+repository](https://github.com/greenelab/scrum). An issue is
+automatically created for each day the office is open. These issues
+can be found [here](https://github.com/greenelab/scrum/issues). The
+update should include the following:
+
+    1. What specific item(s) he/she accomplished yesterday.
+    2. What specific item(s) he/she plans to accomplish today.
+    3. Who, if anyone, is blocking him/her?
+    4. Who, if anyone, is he/she blocking?
```

Comment:
It occurs to me that I changed most of the document because of the line length fixes, so I'm going to highlight the three places I actually changed the content. This is the first.